src/components/footer.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { LifeOmic } from '../components/icons/lifeomic';
+
+export const Footer = ({ pkgVersion }) => {
+  return (
+    <>
```

Comment:
You should be able to remove this fragment, since you are returning a top-level `footer` tag 👍